### PR TITLE
Ell binning and plots

### PR DESCRIPTION
**Pull request by jpratmarti**
Adding the following features:
- Possibility to easily change the ell binning. Added the following options in the config file in `twopoint_fourier.py`:
        "ell_min": 100,
        "ell_max": 1500,
        "n_ell": 20,
        "ell_spacing": 'log'
- Switched the plotting script to load `summary_statistics_fourier.sacc` instead of `twopoint_data_fourier.sacc` to include errorbars in the plots (the covariance is now in the `summary_statistics_fourier.sacc` file).

### CODE REVIEW

File: txpipe/twopoint_fourier.py [hunk 1]
```diff
@@ -439,11 +446,16 @@ def choose_ell_bins(self, pixel_scheme, f_sky):
...
+        '''
+        
         # The subclass of NmtBin that we use here just adds some
         # helper methods compared to the default NaMaster one.
         # Can feed these back upstream if useful.
-        ell_bins = MyNmtBin(int(pixel_scheme.nside), nlb=nlb)
+
+        # Creating the ell binning from the edges using this Namaster constructor.
+        edges = np.geomspace(self.config['ell_min'], self.config['ell_max'], self.config['n_ell'])
+        edges_int = edges.astype(int)
```
Comment (damonge) [inline txpipe/twopoint_fourier.py:457]: The problem with this is that sometimes, after casting to `int`, the result of `geomspace` will have repeated values. What I usually do is:
```
edges = np.unique(np.geomspace(lmin, lmax, n_ell).astype(int))
```

**Review by damonge — COMMENTED** [review]
Hi @jpratmarti 

looks good, just a tiny comment on repeated ell values.

Also, I see you guys are using the `MyNmtBin` wrapper. Is there anything missing in the standard `NmtBin`s that you'd like to have (e.g. there's already a `from_edges` class method)? If there is, I'd be happy to implement it and then switch to them it in a separate PR.

Also, it seems there are conflicts with master, so you should sort those out ;-)

**Comment by joezuntz** [comment]
@damonge I'll do a PR on namaster with the changes we have here if that would be welcome.
Joe

**Comment by damonge** [comment]
Yes, sure! Thanks @joezuntz

File: txpipe/twopoint_fourier.py (hunk 1 above)
Comment (jpratmarti) [reply txpipe/twopoint_fourier.py]: Great thanks a lot David for reviewing this. I added your solution to the repeating values, and will now look at the conflicts with master.

**Review by damonge — APPROVED** [review]
LGTM!

**Comment by joezuntz** [comment]
Two more quick things - could you remove the old `bandwidth` option that no longer does anything, and either implement the log-spacing option or remove it?

**Pull request merged**